f8a_worker/workers/recommender.py

```diff
@@ -639,6 +642,78 @@ def _get_stack_values(self, similar_stacks_list):
             similarity_list.append(s_stack)
         return similarity_list
 
+def invoke_license_analysis_service(user_stack_packages, alternate_packages, companion_packages):
+    license_url = LICENSE_SCORING_URL_REST + "/api/v1/stack_license"
+
+    payload = {
+        "packages": user_stack_packages,
+        "alternate_packages": alternate_packages,
+        "companion_packages": companion_packages
+    }
+
+    resp = {}
+    try:
+        license_req = get_session_retry().post(license_url, data=json.dumps(payload))
+        resp = license_req.json()
+    except:  # TODO customized exception
+        msg = traceback.format_exc()
+        _logger.error("Unexpected error happened while invoking license analysis!\n{}".format(msg))
```

Comment:
I think `_logger.exception("Unexpected error happened while invoking license analysis!")` should do the same, see https://stackoverflow.com/questions/5191830/best-way-to-log-a-python-exception

Reply:
Done. I tested locally and confirmed that `_logger.exception()` works as expected.